quests/variables_and_templates.md

```diff
@@ -71,32 +71,28 @@ references to the `/etc/pasture_config.yaml` configuration filepath with the
 variable.
 
 ```puppet
-class pasture {
-
-  $port                = '80'
-  $default_character   = 'sheep'
-  $pasture_config_file = '/etc/pasture_config.yaml'
-
-  package { 'pasture':
+class pasture (
+  $port                = '80',
```

Comment:
The text above talks about variables. I think you haven't introduced parameters yet.